docs/scos/dev/tutorials-and-howtos/howtos/howto-import-big-databases-between-environments.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "HowTo: Import big databases between environments"
+description: Learn how you can relatively quickly import big data between your environments
+last_updated: April 5, 2023
+template: howto-guide-template
+---
+
+Suppose you have two testing environments, and you need to migrate a large amount of data from one environment to another to perform different tests with the same data. If you have little data, you can export it using the `mysqldump` command. However, for large amounts of data, this method can be slow due to long waiting time and VPN connection issues. In this case, you can use AWS S3 to efficiently import the data between the environments. Here's how to do it:
+
+1. Go the the Jenkins instance of the environment from where you want to import the data. 
+2. Export the database as a compressed file and upload it to an S3 bucket using the following command:
+
+```php
+mysqldump -h $SPRYKER_DB_HOST -u $SPRYKER_DB_ROOT_USERNAME -p $SPRYKER_DB_PASSWORD $SPRYKER_DB_DATABASE | gzip | aws s3 cp - s3://your_bucket_name/backup.$(date +"%Y-%m-%d__%H-%M-%S").sql.gz
+```
+
+![mysqldump-command-in-jenkins](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/dev/tutorials-and-howtos/howtos/howto-import-big-databases-between-environments/mysqldump-command-in-jenkins.png)
+
+3. Monitor the command execution until it finishes. Once it's finished, go to AWS and download the dump file to the necessary environment. Since the S3 bucket is shared, the dump file is accessible from any of your environments.
+4. To recover the dump file, run the following command:
+
+```php
+mysql -u $SPRYKER_DB_ROOT_USERNAME -p $SPRYKER_DB_PASSWORD $SPRYKER_DB_DATABASE < dumpfile.sql.gz
```

Comment:
We can use the following command to recover the backup.
```
zcat your_database_dump.sql.gz | mysql -u $SPRYKER_DB_ROOT_USERNAME -p $SPRYKER_DB_PASSWORD $SPRYKER_DB_DATABASE
```

If the backup is compressed, then we need to uncompress it first.

Reply:
I've updated this step